### PR TITLE
Filter out invalid tweets

### PR DESCRIPTION
**Pull request by scifantastic**
Also a couple of small logging changes. I can revert those if you want.

### CODE REVIEW

**Review by gitphantomman — COMMENTED** [review]
Thanks. I wanna same thing for reddit.

**Review by gitphantomman — COMMENTED** [review]
And can you let me know what is the small logging changes?

File: neurons/validator.py [hunk 1]
```diff
@@ -277,7 +285,7 @@ def main( config ):
                     # set all nodes without ips set to 0
                     scores = scores * torch.Tensor([metagraph.neurons[uid].axon_info.ip != '0.0.0.0' for uid in metagraph.uids])
             # Periodically update the weights on the Bittensor blockchain.
-            if step % 4 == 0:
+            if step % 4 == 2:
                 bt.logging.info(f"\033[92m ᕕ ⏩ Sending reddit query. \033[0m")
```
[the remark this answers is not in the document]
Comment (scifantastic) [reply neurons/validator.py:280]: Done

**Review by gitphantomman — APPROVED** [review]
Great. Looks good. Just merged.

**Pull request merged**